theurgy.lic

```diff
@@ -167,7 +211,9 @@ until ['flawless performance to those on high', 'In your condition'].include? DR
   DRC.fix_standing
 end
 
-fput "get #{$settings.flint_lighter}", 'You get a', 'You are already'
+while checkright != "#{$settings.flint_lighter}"
+  fput "get #{$settings.flint_lighter}", 'You get a', 'You are already'
+end
 get_from_container('incense')
```

Comment:
added this while around the get flint_lighter because theurgy (not often) but sometimes bugs out on me here.  It somehow was missing grabbing this (would get ...wait twice or so and it would move on for some reason) and then spam the room forever with trying to light incense with no lighter.  I thought this would be a simple way to keep it in that spot until it is sure you have it.